lib/bloc/contact_interface_bloc.dart

```diff
@@ -0,0 +1,6 @@
+import '../model/contact.dart';
+import 'bloc.dart';
+
+abstract class ContactInterfaceBloc extends Bloc {
```

Comment:
the name convention for the interfaces is i_name_of_the_class. In this case it should be i_contact_bloc. This is necessary for the name files and the class name. Apply it wherever is necessary.

Reply:
done